.github/workflows/azure-static-web-apps-CI.yml

```diff
@@ -5,25 +5,34 @@ on:
     branches:
       - "**"
       - "!main"
+  pull_request:
+    branches:
+      - main
 
 jobs:
   build_and_deploy_job:
     runs-on: ubuntu-latest
     name: Build and Deploy Job
     environment: staging
     steps:
+      # Step 1: Check out the code from the repository
```

Comment:
nit. The Step 1, 2, 3, are basically repeating the name of the step which renders the comment a bit redundant. I would leave it out.

Reply:
The step 1,2,3 have been removed